_guides/client/basic.md

```diff
@@ -11,6 +11,16 @@ extern crate hyper;
 extern crate tokio_core;
 ```
 
+Make sure you have these dependencies listed in your ```Cargo.toml```, else you may get errors. You may need 
```

Comment:
I'd probably put this before the external crate part. And just say something like "We need to tell Cargo our dependencies first, by having this in the Cargo.toml."